.github/workflows/jacoco_report.yml

```diff
@@ -62,8 +60,8 @@ jobs:
           paths: >
             ${{ github.workspace }}/core/target/scala-${{ env.SCALA_SHORT_VERSION }}/jacoco/report/jacoco.xml
           token: ${{ secrets.GITHUB_TOKEN }}
-          min-coverage-overall: ${{ env.COVERAGE_OVERALL_EXPECTATION }}
-          min-coverage-changed-files: ${{ env.COVERAGE_CHANGED_EXPECTATION }}
+          min-coverage-overall: ${{ 57.0 }}
```

Comment:
Are {{ }} still needed if inside is a number?
I believe we talked about removing these constants, right?

Reply:
Here, I pointed to 1st location where I saw it. This is on more locations in yml file.

Reply:
Completely rewritten.